src/dune_pkg/solver_env.mli

```diff
@@ -38,6 +38,24 @@ module Variable : sig
     end
   end
 
+  module User : sig
+    (** A custom variable that can be configured by a user *)
+    type t
```

Comment:
Shouldn't this be the same as `Package_variable.Name.t`?

Reply:
Yes you're right. Updated.